torch/_prims/executor.py

```diff
@@ -1,17 +1,22 @@
-# mypy: allow-untyped-defs
-from typing import Callable, Optional
+from typing import Any, Callable, Optional, TypeVar
+from typing_extensions import ParamSpec
 
 from torch._prims.context import TorchRefsMode
 from torch.fx import GraphModule
 from torch.fx.experimental.proxy_tensor import make_fx, wrapper_and_args_for_make_fx
 
 
+T = TypeVar("T")
+P = ParamSpec("P")
+Args = TypeVar("Args")
+
+
 def execute(
     gm: GraphModule,
-    *args,
+    *args: object,
```

Comment:
I think this could be a typing_extensions.TypeVarTuple to forward all the args to the next function.